src/foam/u2/UnstyledActionView.js

```diff
@@ -93,6 +93,7 @@ foam.CLASS({
 
   methods: [
     function initE() {
+      this.attrs({name: this.action.name});
```

Comment:
This should be moved inside the `if ( this.action ) { ... }` block below so we know the action is set.

Also, our style convention is to use spaces inside of objects written on a single line, like this: `{ name: this.action.name }`.